app/src/Account.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from "react";
```

Comment:
useState is not used currently so its throwing me some warnings in the console~~ but im sure you will use it later but just a reminder :)!

Reply:
Thanks for the heads up!